test/examples/common/features/create_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'spec_helper'
+
+describe 'rake es:create', type: :cli do
+
+  before { system 'rake es:drop' }
+
+  def execute!
+    system 'rake es:create'
+  end
+
+  context 'when run in test environment' do
+    let(:environment) { 'test' }
+
+    it 'creates and migrates test database' do
```

Comment:
In this mode it should not create a database?

Reply:
Hmm.. does common mean using a shared database, or are these tests common between the shared and separate tests?